sirepo/pkcli/runner.py

```diff
@@ -131,7 +131,7 @@ def status(self, jid, run_dir):
                 return
             try:
                 env = _subprocess_env()
-                with open(run_dir / template_common.RUN_LOG, "a+b") as run_log:
+                with open(run_dir / template_common.RUN_LOG, 'a+b') as run_log:
```

Comment:
Overloaded `/` is not as readable as run_dir.join(template_common.RUN_LOG). Moreover, if the reader has to ask the question: what does `/` do, they can't search on [py path /](https://www.google.com/search?q=py+path+/) to get the answer whereas [py path join](https://www.google.com/search?q=py+path+join) produces the answer right on the google search page, because google can understand the question.  If google can't understand the question, that's a clue that the API has a bad name.

In general, avoid using or implementing overloaded operators like this. The `/` is a "cute" but cute is rarely maintainable. The use of `/` contradicts the meaning of `/` as divide: smaller result comes from a divide, is this removing or splitting something? This creates an unnecessary cognitive load. The word "join" is used in its proper English usage: connecting two things.